odoo/tools/image.py

```diff
@@ -262,16 +263,16 @@ def colorize(self):
         return self
 
 
-def image_process(base64_source, size=(0, 0), verify_resolution=False, quality=0, crop=None, colorize=False, output_format=''):
+def image_process(base64_source, size=(0, 0), verify_resolution=False, quality=0, crop=None, colorize=False, output_format='', user_active=True):
     """Process the `base64_source` image by executing the given operations and
     return the result as a base64 encoded image.
     """
-    if not base64_source or ((not size or (not size[0] and not size[1])) and not verify_resolution and not quality and not crop and not colorize and not output_format):
+    if not base64_source or ((not size or (not size[0] and not size[1])) and not verify_resolution and not quality and not crop and not colorize and not output_format and user_active):
         # for performance: don't do anything if the image is falsy or if
         # no operations have been requested
         return base64_source
 
-    image = ImageProcess(base64_source, verify_resolution)
+    image = ImageProcess(base64_source, user_active, verify_resolution)
```

Comment:
`ImageOps.grayscale(image)`  should probably be done here, it is not linked to the `base64_to_image` function.
And param name be something related to grayScale and not user... Here your are in tools...

`image_base64 = image_process(image_base64, size=(int(width), int(height)), crop=crop, quality=int(quality), grayscale=not user_active)` for example